### ISSUE TITLE
Update versions via a bot that will actually trigger CI

### ISSUE THREAD

**Issue by purcell** [purcell/nix-emacs-ci#228]
Currently the automatically-produced version update PRs need to be closed then re-opened to trigger CI.

Apparently a renovate config like this would cause CI to run automatically, so I should consider switching:

https://github.com/domenkozar/homelab/blob/main/.github/renovate.json

**Comment by ncouture** [comment]
Are you familiar with release-please? In my experience it's the best release automation tool out there and you can trigger CI, and releases, via the release-please CLI.

https://github.com/googleapis/release-please

**Comment by akirak** [comment]
@ncouture Thanks, but I don't think the tool is relevant here. We periodically update dependencies and rebuild the packages, but the repository itself is seldom tagged. It's mostly about updating dependencies.

An ideal solution would be to update specific inputs of the Nix flake (which Renovate doesn't support yet), create a PR, and merge the PR if everything is successfully built. There are some hacks for the create-pull-request action to automatically trigger CI, but I don't know if @purcell would want to adopt any of them at the cost of degraded security. I am fine with the current process of maintenance.

**Comment by purcell** [comment]
Agree, renovate or similar would be an alternative to the current set-up, but I think the current compromises are fine. Thanks for reading this and dropping the link, though, @ncouture!
